src/solver/main_solver.cpp

```diff
@@ -32,9 +32,12 @@ int main()
 
 /*******************************************************************************************************************************************/
 
+    nlohmann::json json_file;
+    std::ifstream input_file("./sample_instances/sample.json");
```

Comment:
How is the error treated, if the file is not existing?

Reply:
This is dummy code, for now. I agree that the input file must be pass as an argument and safely processed by the program, but I am planning to do that in a future PR that would be exclusively for safe parsing of the input.